### PR TITLE
Add reference to maven36 image and replace some references to 'slave'…

### PR DESCRIPTION
**Pull request by akram**
… by 'agent'

### CODE REVIEW

**Comment by akram** [comment]
can you PTAL @Preeticp  ?

**Comment by akram** [comment]
This is related to this: https://bugzilla.redhat.com/show_bug.cgi?id=1924811

File: using_images/other_images/jenkins.adoc [hunk 1]
```diff
@@ -429,7 +429,7 @@ container hosts using Kubernetes and {product-title}.
 
 To use the Kubernetes plug-in, {product-title} provides five images suitable
 for use as Jenkins agents: the *_Base_*, *_Maven_*, and *_Node.js_* images. See
-xref:jenkins_slaves.adoc#overview[Jenkins Agents] for more information.
+xref:jenkins_agents.adoc#overview[Jenkins Agents] for more information.
 
 NOTE:  the jenkins-slave-maven-* and jenkins-slave-nodejs-* images are being
```
Comment (Preeticp) [inline using_images/other_images/jenkins.adoc:434]: Not part of this PR, but I wonder if you could change 'the' in this note to 'The', please?

**Review by Preeticp — COMMENTED** [review]
@akram thanks for doing this. I have suggested some minor changes, otherwise it looks good to me.

**Comment by Preeticp** [comment]
@bmcelvee can you take a look too, please?

**Comment by bmcelvee** [comment]
LGTM! @Preeticp I updated the redirects register too. 

@akram it's outside of the scope of this PR, but is it safe to update the references in 4.x to "agent" too? I checked a while back and there were some lingering instances to "slave" in the code. If we're clear to make those changes, we can open an issue to track that work (@Preeticp I'd be happy to help there). Thanks!
